modules/gitops-disable-dex-using-spec-sso.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-using-dex
+
+:_content-type: PROCEDURE
+[id="gitops-disable-dex-using-spec-sso_{context}"]
+= Enabling or disabling Dex using .spec.sso
+
+You can configure {gitops-title} to use Dex as its SSO authentication provider by setting the `.spec.sso` parameter.
+
+.Procedure
+
+. To enable Dex, set the `.spec.sso.provider: dex` parameter in the YAML resource of the Operator:
+
++
+[source,yaml]
+----
+spec:
```

Comment:
```suggestion
----
...
spec:
```
Please use ellipses to indicate where you have truncated content from the code block. You might also consider including the few lines that appear above this spec because it helps the user identify which YAML resource you are mentioning.

Reply:
See `modules/adding-a-custom-logo.adoc`

```
. Edit the web console's Operator configuration to include `customLogoFile` and `customProductName`:
+
[source,terminal]
----
$ oc edit consoles.operator.openshift.io cluster
----
+
[source,yaml]
----
apiVersion: operator.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec:
  customization:
    customLogoFile:
      key: console-custom-logo.png
      name: console-custom-logo
    customProductName: My Console
----
```